app/controllers/items_controller.rb

```diff
@@ -11,10 +12,37 @@ def create
     redirect_to current_user
   end
 
+  def destroy
+    @item = current_user.items.find(params[:id])
+    @item.update_attribute(:completed_at, Time.now)
+    if @item.destroy
+      flash[:notice] = "Item was deleted."      
+    else
+      flash.now[:alert] = "There was an error deleting your item. Please try again."      
+    end
+
+    respond_to do |format|
+      format.html
+      format.js
+    end    
+
+  end
+
+  def complete
```

Comment:
So unless you plan on keeping the completed items around with a status of completed or something, this method is not necessary.